docker/nginx/redirects-301.map

```diff
@@ -1,6 +1,7 @@
 ~^(.*/[^.^#^/]*[^/])$ $1/;
 ~^/1.7/.+ /1.7/;
 ~^/1.8/.+ /1.8/;
+~^/ksphere/(.*)$ /dkp/$1;
```

Comment:
Good idea.

Reply:
i figured all you'd need to do is rename the pages folder from `pages/ksphere` to `pages/dkp` and then add in this redirect. Wouldn't all the ksphere links redirect to dkp path based on this rule?

Reply:
if we also changed the tooling-related things and the "later" redirect rules, that *would* work of course.
i think changing the links themselves is a good practice though and does not seem very risky. every client saves the redirect turnaround time and we can reason about how our internal links work more easily in a couple month from now when everyone has forgotten that this change happened. :)

i'd be open to compelling reasons not to change the links themselves though of course.